### PR TITLE
[permissions] Clarify `Compatibility` permission's description 

### PR DESCRIPTION
**Pull request by Olf0**
… as originally suggested at https://github.com/sailfishos/sailjail-permissions/issues/127#issuecomment-1207232462.

### CODE REVIEW

**Review by llewelld — APPROVED** [review]
I'm not best-placed to judge (so it'd be good to get a second opinion), but for me this updated wording looks clear and matches my understanding of the permission, so this LGTM.

**Comment by attah** [comment]
A bit of a nitpick... but since we are already here.

File: README.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ Internal permissions that applications generally should not use directly:
 | :---       | :---        |
 | Base       | Base set of permissions that every application is granted implicitly. |
 | CaptivePortal |
-| Compatibility | Access to binaries for compatibility. Direct use of binaries should be avoided. Please note that this permission is provided in order to support old applications, new applications should try to use other means. The binaries may disappear in any release, for any reason, but especially if we provide APIs with similar functionality. |
+| Compatibility | Allows for accessing certain binaries to provide some degree of compatibility for extant apps. In general, only library classes and functions are allowed to be used by new applications. The access to these binaries may disappear in any Sailfish&nbsp;OS release, especially when official APIs with similar functionality are provided. |
```
Comment (Thaodan) [inline README.md:202]: I'm not sure on the word "official", API aren't less official when they not intended to be used.
Most of these provide binaries in the compatibility permissions are those that were used to workaround Harbour restrictions.
I would remove the word official and keep the wording of the official sentence.

**Review by Olf0 — COMMENTED** [review]
@llewelld, a soft "ping". :wink:

**Pull request merged**